### PR TITLE
Update to new reminders API

### PR DESCRIPTION
**Pull request by TomPretty**
## What are you doing in this PR?
Update the TY page set reminder component to use the new reminders API ([trello card](https://trello.com/c/P2olSwqR/2486-reminders-architecture-v2)).

### request
Screenshot 2021-02-12 at 11 46 29

### cloudwatch logs
Screenshot 2021-02-12 at 11 50 03

### CODE REVIEW

File: support-frontend/conf/CODE.public.conf [hunk 1]
```diff
@@ -17,7 +17,7 @@ paymentApi.url="https://payment.code.dev-guardianapis.com"
...
...ywhere.co.uk https://stripe-intent-code.support.guardianapis.com www.paypalobjects.com t.paypal.com/ members-data-api.code.dev-theguardian.com metric-push-api-code.support.guardianapis.com www.paypal.com www.sandbox.paypal.com js.stripe.com https://payments-sandbox.amazon.com https://api-cdn.amazon.com static-na.payments-amazon.com pasteup.guim.co.uk ophan.theguardian.com j.ophan.co.uk media.guim.co.uk uploads.guim.co.uk www.google-analytics.com www.googletagmanager.com tagmanager.google.com polyfill.io www.googleadservices.com googleads.g.doubleclick.net www.google.com www.google.co.uk optimize.google.com static.ads-twitter.com bat.bing.com bid.g.doubleclick.net t.co analytics.twitter.com stats.g.doubleclick.net www.youtube-nocookie.com connect.facebook.net www.facebook.com consumer.krxd.net cdn.krxd.net beacon.krxd.net secure.adnxs.com checkout.stripe.com fonts.googleapis.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com sentry.io www.dwin1.com data: wss: 'unsafe-inline' q.stripe.com payment.code.dev-guardianapis.com https://interactive.guim.co.uk/ https://www.theguardian.com/ https://theguardian.com/ https://contribution-referrals-code.support.guardianapis.com/referral-code"
```
Comment (TomPretty) [inline support-frontend/conf/CODE.public.conf:20]: hard to see from diff, but this just removed `https://contribution-reminders-code.support.guardianapis.com/` from the list

File: support-frontend/conf/DEV.public.conf [hunk 2]
```diff
@@ -17,7 +17,7 @@ paymentApi.url="https://payment.code.dev-guardianapis.com"
...
...e-intent-code.support.guardianapis.com http://localhost:9000 members-data-api.thegulocal.com metric-push-api-code.support.guardianapis.com www.paypalobjects.com www.paypal.com t.paypal.com www.sandbox.paypal.com js.stripe.com https://payments-sandbox.amazon.com https://api-cdn.amazon.com static-na.payments-amazon.com pasteup.guim.co.uk ophan.theguardian.com j.ophan.co.uk media.guim.co.uk uploads.guim.co.uk www.google-analytics.com www.googletagmanager.com tagmanager.google.com polyfill.io www.googleadservices.com googleads.g.doubleclick.net www.google.com www.google.co.uk optimize.google.com static.ads-twitter.com bat.bing.com bid.g.doubleclick.net t.co analytics.twitter.com stats.g.doubleclick.net www.youtube-nocookie.com connect.facebook.net www.facebook.com consumer.krxd.net cdn.krxd.net beacon.krxd.net secure.adnxs.com checkout.stripe.com fonts.googleapis.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com sentry.io www.dwin1.com data: wss: 'unsafe-inline' 'unsafe-eval'  q.stripe.com payment.code.dev-guardianapis.com https://interactive.guim.co.uk/ https://www.theguardian.com/ https://theguardian.com/ https://contribution-referrals-code.support.guardianapis.com/referral-code"
```
Comment (TomPretty) [inline support-frontend/conf/DEV.public.conf:20]: this added `https://support.code.dev-theguardian.com/` and removed `https://contribution-reminders-code.support.guardianapis.com/`

File: support-frontend/conf/PROD.public.conf [hunk 3]
```diff
@@ -16,7 +16,7 @@ paymentApi.url="https://payment.guardianapis.com"
...
...e-intent.support.guardianapis.com members-data-api.theguardian.com metric-push-api-prod.support.guardianapis.com www.paypalobjects.com www.paypal.com t.paypal.com www.sandbox.paypal.com js.stripe.com https://payments.amazon.com https://payments-sandbox.amazon.com https://coin.amazonpay.com https://api-cdn.amazon.com static-na.payments-amazon.com pasteup.guim.co.uk ophan.theguardian.com j.ophan.co.uk media.guim.co.uk uploads.guim.co.uk www.google-analytics.com www.googletagmanager.com tagmanager.google.com polyfill.io www.googleadservices.com googleads.g.doubleclick.net www.google.com www.google.co.uk optimize.google.com static.ads-twitter.com bat.bing.com bid.g.doubleclick.net t.co analytics.twitter.com stats.g.doubleclick.net www.youtube-nocookie.com connect.facebook.net www.facebook.com consumer.krxd.net cdn.krxd.net beacon.krxd.net secure.adnxs.com checkout.stripe.com fonts.googleapis.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com sentry.io www.dwin1.com data: wss: 'unsafe-inline' q.stripe.com payment.guardianapis.com https://interactive.guim.co.uk/ https://www.theguardian.com/ https://theguardian.com/ https://contribution-referrals.support.guardianapis.com/referral-code"
```
Comment (TomPretty) [inline support-frontend/conf/PROD.public.conf:19]: this removed `https://contribution-reminders.support.guardianapis.com/`

**Review by tomrf1 — APPROVED** [review]
great work!

**Pull request merged**